components/formats-api/src/loci/formats/in/DefaultMetadataOptions.java

```diff
@@ -32,27 +32,21 @@
 
 package loci.formats.in;
 
-import loci.formats.FormatOptions;
-
 
 /**
  * Default implementation of {@link loci.formats.in.MetadataOptions}.
  */
-public class DefaultMetadataOptions
-    extends FormatOptions implements MetadataOptions {
-
-  public static final String METADATA_LEVEL_KEY = "metadata.level";
-  public static final MetadataLevel METADATA_LEVEL_DEFAULT = MetadataLevel.ALL;
+public class DefaultMetadataOptions implements MetadataOptions {
 
-  public static final String READER_VALIDATE_KEY = "reader.validate.input";
-  public static final boolean READER_VALIDATE_DEFAULT = false;
+  private MetadataLevel metadataLevel;
+  private boolean validate;
 
   /**
    * Construct a new {@code DefaultMetadataOptions}. Set the metadata level
    * to {@link #METADATA_LEVEL_DEFAULT} and disable file validation.
```

Comment:
As per https://ci.openmicroscopy.org/view/Failing/job/BIOFORMATS-DEV-merge-build/468/warnings40Result/ this needs to be updated as well.